src/geo/transform.js

```diff
@@ -146,7 +171,7 @@ class Transform {
      *
      * @private
      */
-    getVisibleWrappedCoordinates(tileCoord) {
+    getVisibleWrappedCoordinates(tileCoord: {x: number, y: number, z: number}) {
```

Comment:
Is there an advantage to being explicit like this instead of using the `TileCoord` type?